wsi/drm/drmdisplay.cpp

```diff
@@ -571,6 +573,48 @@ void DrmDisplay::ApplyPendingLUT(struct drm_color_lut *lut) const {
   drmModeDestroyPropertyBlob(gpu_fd_, lut_blob_id);
 }
 
+uint64_t DrmDisplay::DrmRGBA(uint32_t bpc, uint16_t red, uint16_t green,
+			      uint16_t blue, uint16_t alpha) const {
+  int shift;
+  uint64_t result;
+
+  if (bpc > 16)
```

Comment:
bpc cannot just be a multiple of 8; for instance, it can be 10. In other words, there are color formats that expect to have 10 bits per color component.